pkg/dogstatsd/listeners/packets_buffer.go

```diff
@@ -6,10 +6,22 @@
 package listeners
 
 import (
+	"expvar"
 	"sync"
 	"time"
 )
 
+var (
+	packetsBufferExpvars = expvar.NewMap("PacketsBuffer")
+	packetsDropped       = expvar.Int{}
+	packetsProcessed     = expvar.Int{}
```

Comment:
Can you add this kind of telemetry instead? https://github.com/DataDog/datadog-agent/pull/5188/files#diff-22cc276d8141eef6c4812334db7b9a70

Reply:
sure, done